applications/crossbar/doc/ledgers.md

```diff
@@ -0,0 +1,86 @@
+/*
+Section: Crossbar
+Title: Ledgers
+Language: en-US
```

Comment:
Add `Version: 3.22` and keep these updated with other docs so people know what version the docs refer to when they look at features in the doc.